nixos/modules/programs/openlinkhub.nix

```
@@ -0,0 +1,52 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.openlinkhub;
+  pkg = config.programs.openlinkhub.package;
+in
+{
+  options.programs.openlinkhub = {
+    enable = lib.mkEnableOption "openlinkhub";
+    package = lib.mkPackageOption pkgs [ "delta" "openlinkhub" ] { };
+  };
+
+  config = lib.mkIf cfg.enable {
+    users.users.openlinkhub = {
+      isSystemUser = true;
+      group = "openlinkhub";
+    };
+    users.groups.openlinkhub = { };
+
+    systemd.services.OpenLinkHub = {
```

I suppose it's not a hard rule, but I very rarely come by systemd services with capital letters. Only ever lowercase and dashes. Consider lowercasing it, if only to reduce surprise for downstream users. But feel free to mark as resolved if you think it should stay the way it is.

```suggestion
    systemd.services.openlinkhub = {
```